enhancements/cluster-logging/multi-cluster-log-forwarder.md

```diff
@@ -0,0 +1,262 @@
+
+* The **cluster-logging-operator** is deployed to the *openshift-logging* namespace
+* The **cluster-logging-operator** is able to watch any namespace
+
+#### Multiple-Instance Mode: Allowing multiple ClusterForwarder and ClusterLogging resources 
+
+This workflow supports any ClusterLogForwarder except one named "instance" in the *openshift-logging namespace*.  The resource openshift-logging/instance is significant to supporting the legacy workflow.
+
+**NOTE:** Vector is the only supported collector implimentation in this mode.
+
+**cluster administrator** is a user:
+
+* responsible for maintaining the cluster
+* able to bind cluster roles to serviceaccounts  
+* that deploys the **cluster-logging-operator**
+
+**namespace administrator** is a user:
+
+* able to create a serviceaccount
+* able to create a serviceaccount token
+* manages a **ClusterLogForwarder** custom resource
+
+The general workflow:
+
+* The namespace administrator creates a service account to be used by a log collector.  The service account must additionally include a token if there is intent to write to log storage that depends upon a token for authentication.
+* The cluster administrator binds cluster roles to the service account for the log types they are allowed to collect (e.g. audit, infrastructure).  Several roles are added to the operator manifest and look something like:
+
+```yaml
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: collect-audit-logs
+    rules:
+    - apiGroups:
+      - "logging.openshift.io"
+      resources:
+      - logs
+      resourceNames:
+      - audit
+      verbs:
+      - collect
+
+```
+This role allows collection of application logs and requires the namespace administor to bind the service account to the role like:
+
+```text
+    oc create clusterrolebinding kube-audit-log-collection --clusterrole=collect-audit-logs --serviceaccount=openshift-kube-apiserver:audit-collector-sa
+```
+
+* The namespace administrator creates a **ClusterLogForwarder** CR that references the serviceaccount and the inputs for which that serviceaccount is allowed to collect
+
+```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: ClusterLogForwarder
+    metadata:
+      name: audit-collector
+      namespace: openshift-kube-apiserver
+    spec:
+      serviceAccount: audit-collector-sa
+      pipelines:
+       - inputRefs:
+         - audit
+         outputRefs:
+         - loki
+      outputs:
+      - name: loki
+        type: loki
+        url: https://mycenteralizedserver.some.place
+```
+
+##### Use of ClusterLogging resource
+This resource is optional in multiple instance mode and is a departure from the legacy mode where a ClusterLogging resource is always required with a ClusterLogForwarder. A namespace administrator must define a **ClusterLogging** CR named the same as the **ClusterLogForwarder** CR and in the same namespace when needing to spec collector resources or placement.
```

Comment:
One small improvement I could see: allow for an optional spec field to allow ClusterLogging CR to reference a specifically named ClusterLogForwarding CR to not always rely on them being the same name

Reply:
We talked about this but @alanconway preferred to rely upon naming convention instead